### DISCUSSION TITLE
how to use annotationProcessor or plugin or compile in Sketchware any idea?

### DISCUSSION THREAD

**Discussion by AndroXStudiO** [Sketchware-Pro/Sketchware-Pro#567] [Q&A] [answered]
how to use annotationProcessor or plugin or compile in Sketchware any idea?

> Please help me if any alternative way to add
 
implementation 'com.jakewharton:butterknife:10.2.3'
  annotationProcessor 'com.jakewharton:butterknife-compiler:10.2.3'

**Comment by j-px** [comment, marked as answer]
Sketchware and Sketchware Pro don't use Gradle to build projects. Annotation processors that way, (and the implementation, include, compile, etc. "keywords" too) are Gradle features, and are unavailable in Sketchware and Sketchware Pro.

**Reply by AndroXStudiO** [reply]
Okey fine.  but any alternative way?

**Reply by j-px** [reply]
Nope. Not so far, at least. It's theoretically possible to add.
